desktop-exporter/app/components/detail-view/events-panel.tsx

```diff
@@ -26,7 +26,7 @@ function EventItem(props: EventItemProps) {
   let timeSinceSpanStart = getDurationNs(spanStartTime, event.timestamp);
   let durationString = getDurationString(timeSinceSpanStart);
   let eventAttributes = Object.entries(event.attributes).map(([key, value]) => (
-    <li key={key + value}>
+    <li key={key + value?.toString}>
```

Comment:
I think `value?.toString` will either be null or a reference to a function?

```suggestion
    <li key={key + value?.toString()}>
```

Reply:
Since `key` is always a string, I think value will be coerced to a string, though that may not play well with types